crates/circuit/src/dag_circuit.rs

```diff
@@ -2593,30 +2592,65 @@ def _format(operand):
         Ok(tup.into_any().iter().unwrap().unbind())
     }
 
-    /// Iterator for edge values and source and dest node
+    /// Iterator for edge values with source and destination node.
     ///
-    /// This works by returning the output edges from the specified nodes. If
+    /// This works by returning the outgoing edges from the specified nodes. If
     /// no nodes are specified all edges from the graph are returned.
     ///
     /// Args:
-    ///     nodes(DAGOpNode, DAGInNode, or DAGOutNode|list(DAGOpNode, DAGInNode, or DAGOutNode):
+    ///     nodes(DAGOpNode, DAGInNode, DAGOutNode or Vec of DAGOpNode, DAGInNode, or DAGOutNode:
```

Comment:
Since we're documenting a `pymethod` here, we ought to keep these types as Python, since this docstring will get attached to the Python docs for `DAGCircuit` 🙂.

For Rust-only methods (the methods outside of the `pyclass` impl block), we're free to use Rust types.